addons/uom/static/src/js/uom_field_widget.js

```diff
@@ -0,0 +1,147 @@
+odoo.define('uom.UomFieldWidget', function (require) {
+"use strict";
+
```

Comment:
also, new code is now written in esnext (we can use everything that is supported by all evergreen browsers)